code/datums/mind.dm

```diff
@@ -44,7 +44,8 @@
 		msg_admin_niche("[key]/[ckey] has tried to transfer to deleted [new_character].")
 		return
 
-	SEND_SIGNAL(current.client, COMSIG_CLIENT_MIND_TRANSFER, new_character)
+	if(current.client)
+		SEND_SIGNAL(current.client, COMSIG_CLIENT_MIND_TRANSFER, new_character)
```

Comment:
Storing signals on the client is a bad idea, client datums get completely reconstructed.

Store this on something more persistent, like the mind datum.